bytes/bytes/api/router.py

```diff
@@ -190,6 +191,11 @@ def create_raw(
                 raw_data=RawDataMeta(id=raw_id, boefje_meta=raw_data.boefje_meta, mime_types=raw_data.mime_types),
             )
             event_manager.publish(event)
+        except pika.exceptions.AMQPError as error:
+            logger.exception("Error sending 'new raw data' event to RabbitMQ")
+            raise HTTPException(
```

Comment:
Is this something we want to expose to someone using the API?

Reply:
I'd say, yes? Its an authenticated user, and their request failed.